packages/edit-site/src/components/routes/use-title.js

```diff
@@ -47,14 +47,7 @@ export default function useTitle( title ) {
 			document.title = formattedTitle;
 
 			// Announce title on route change for screen readers.
-			speak(
-				sprintf(
-					/* translators: The page title that is currently displaying. */
-					__( 'Now displaying: %s' ),
-					document.title
-				),
-				'assertive'
-			);
+			speak( accessibleTitle || formattedTitle, 'assertive' );
```

Comment:
Instead of `accessibleTitle` we could use `title` if we remove the ` ‹ Editor` part and add it later only for the formattedTitle.

Reply:
Done.